Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -4011,7 +4036,10 @@ public IWebElement LocateElementByLocators(ElementInfo currentPOMElementInfo, IL
 
                             else
                             {
-                                targetElement = LocateElementByLocator(locator, ParentContext);
+                                if (ParentContext != null)
+                                {
+                                    targetElement = LocateElementByLocator(locator, ParentContext);
```

Comment:
If element is not shadow root element and just a normal element without parent context, it will not get located?

Reply:
if the element is not a shadow root the initial value of the parent context is Driver so the normal elements will be located